Jenkinsfile

```diff
@@ -14,35 +14,33 @@
 // the License.
 
 // DRYing out the Jenkinsfile...
-build_script = '''
-mkdir -p ${COUCHDB_IO_LOG_DIR}
 
-echo
-echo "Build CouchDB from tarball & test"
```

Comment:
I found these echos easier to grep for in the logfile than the commands themselves. Is there a similar keyword I can search on now?

Reply:
These are separate build stages now, so the log files are entirely separate (at least in the Jenkins UI, I haven't checked the log file uploader).